test/test_datasets.py

```diff
@@ -2126,5 +2126,47 @@ def test_bad_input(self):
                 pass
 
 
+class HD1KTestCase(KittiFlowTestCase):
```

Comment:
Do you subclass here to run some extra tests defined in `KittiFlowTestCase`?

Reply:
Yes, I was a bit lazy and didn't want to write a base class; this seemed overkill considering we're going to deprecate these datasets very soon anyway, and I'm not expecting to add any additional flow dataset ATM.